tests/phpunit/test-wc-rest-stripe-account-keys-controller.php

```diff
@@ -70,10 +70,8 @@ public function test_adding_keys_returns_status_code_200() {
 		$request->set_param( 'webhook_secret', 'webhook-secret-12345' );
 
 		$response = $this->controller->set_account_keys( $request );
-		$expected = [];
 
 		$this->assertEquals( 200, $response->get_status() );
-		$this->assertEquals( $expected, $response->get_data() );
```

Comment:
I had to remove it because now we are returning the account data. 
If we mock the method `get_cached_account_data` and test for the result here, we would already know the outcome, so I don't see the point in it. I'm Leaving only the test for the HTTP status.